src/learn/pages/Voting.jsx

```diff
@@ -7,61 +7,91 @@ import Image from "../../common/components/Image";
 import aiImg from "../../image/aiImg.svg";
 import AttentionMessage from "../../common/components/AttentionMessage";
 import SelectionLabel from "../components/SelectionLabel";
+import Timer from "../../common/components/Timer";
+import TimeUp from "../../common/components/TimeUp";
 
 export default function Voting() {
   const [data, setData] = useState();
-  const [myChoice, setMyChoice] = useState(0);
-  const [attentionMessage, setAttentionMessage] = useState("");
+  const [myChoice, setMyChoice] = useState(-1);
```

Comment:
```suggestion
  const setMyChoice = -1;
```

useStateしてる理由はあるのかいなか

Reply:
letだと送信する際にmyChoiceの値が初期化されてしまいます

Reply:
なのでuseStateを使用しています

Reply:
あれ、どこかでmychoiceが使われてない感じがしたけど使われてる